src/renderer/state/test/mocks.ts

```diff
@@ -421,6 +421,45 @@ export const mockSuccessfulUploadJob: JSSJob = {
   user: "test_user",
 };
 
+export const mockSuccessfulUploadJobWithUnexposedAnnotation: JSSJob = {
+  created: new Date(),
+  currentStage: "Completed",
+  jobId: "123434234",
+  jobName: "mockJob1",
+  modified: new Date(),
+  serviceFields: {
+    files: [
+      {
+        customMetadata: {
+          annotations: [
```

Comment:
I am not familiar with the concept of exposed vs unexposed annotations - what makes these 'unexposed'?

Reply:
[Some annotations are generated ](http://aics.corp.alleninstitute.org/labkey/query/AICS/executeQuery.view?schemaName=filemetadata&query.queryName=Annotation&query.ExposeToFileUploadApp~eq=false)in places other than the FUA, such as in the LabKey plate custom data tab or via MXS, so they're marked to not be "exposed" to the FUA so people don't accidentally use them there. They still get used later down the line in other processing steps / the Explorer app though.
e.g. You don't want people uploading files via the app while marking the "MXS Processing Failed" annotation as true or false -  it wouldn't make sense for the user to declare that either way